core/pom.xml

```diff
@@ -116,11 +116,6 @@ THE SOFTWARE.
       <artifactId>jnr-posix</artifactId>
       <version>3.0.45</version>
     </dependency>
-    <dependency>
-      <groupId>org.kohsuke</groupId>
-      <artifactId>trilead-putty-extension</artifactId>
```

Comment:
Used by several plugins in https://github.com/search?q=org%3Ajenkinsci+PuTTYKey&type=Code , but it should be addressed by Trilead API plugin